src/views/Dashboard/Customer/Home/GeneticAnalysisList.vue

```diff
@@ -0,0 +1,156 @@
+        .genetic-analysis-list__actions
+          ui-debio-icon.genetic-analysis-list__action(
+            :icon="downloadIcon"
+            :class="{ 'genetic-analysis-list__action--disabled': item.status !== 'Done' }"
+            size="16"
+            role="button"
+            stroke
+            stroke-width="4"
+            @click="onDownload(item)"
+          )
+</template>
+
+<script>
+import { medicalResearchIllustration } from "@/common/icons"
+import { downloadIcon } from "@/common/icons"
+import { downloadDecryptedFromIPFS } from "@/common/lib/ipfs"
+import { u8aToHex } from "@polkadot/util"
+import CryptoJS from "crypto-js"
+import Kilt from "@kiltprotocol/sdk-js"
+import DataTable from "@/common/components/DataTable"
+import { mapState } from "vuex"
+
+export default {
+  name: "GeneticAnalysisList",
+  components: { DataTable },
+
+  data: () => ({
+    medicalResearchIllustration,
+    downloadIcon,
+
+    // TODO: Update headers and items to dynamic data from backend later
+    publicKey: null,
+    secretKey: null,
+    headers: [
+      { text: "Service Name", value: "name", sortable: true },
+      { text: "Analyst Name", value: "analyst", sortable: true },
+      { text: "Order Date", value: "createdAt", sortable: true },
+      { text: "Last Update", value: "updatedAt", sortable: true },
+      { text: "Status", value: "status", width: "115", sortable: true },
+      {
+        text: "Action",
+        value: "action",
+        sortable: false,
+        align: "center"
+      }
+    ],
+    items: [
+      {
+        name: "Genome Data",
+        analyst: "Hildegard Einstein",
+        createdAt: "28 Jan 2022",
+        updatedAt: "29 Jan 2022",
+        ipfsLink: "https://ipfs.io/ipfs/QmYaGRJSZV8FjjD9Bd7BAQ5KevCQfbG3mU4yb2PNQAr1Vh/A wonderful.pdf",
+        status: "Inprogress"
+      },
+      {
+        name: "Genotype",
+        analyst: "Sarah Newton",
+        createdAt: "28 Jan 2022",
+        updatedAt: "29 Jan 2022",
+        ipfsLink: "https://ipfs.io/ipfs/QmYaGRJSZV8FjjD9Bd7BAQ5KevCQfbG3mU4yb2PNQAr1Vh/A wonderful.pdf",
+        status: "Done"
+      },
+      {
+        name: "Ancestry DNA Data",
+        analyst: "Jacky Colombus",
+        createdAt: "28 Jan 2022",
+        updatedAt: "29 Jan 2022",
+        ipfsLink: "https://ipfs.io/ipfs/QmYaGRJSZV8FjjD9Bd7BAQ5KevCQfbG3mU4yb2PNQAr1Vh/A wonderful.pdf",
+        status: "Done"
+      },
+      {
+        name: "DCHGenome",
+        analyst: "Azmi Galileo",
+        createdAt: "28 Jan 2022",
+        updatedAt: "29 Jan 2022",
+        ipfsLink: "https://ipfs.io/ipfs/QmYaGRJSZV8FjjD9Bd7BAQ5KevCQfbG3mU4yb2PNQAr1Vh/A wonderful.pdf",
+        status: "Done"
+      },
+      {
+        name: "Ancestry DNA Data B",
+        analyst: "Dr Octopus",
+        createdAt: "28 Jan 2022",
+        updatedAt: "29 Jan 2022",
+        ipfsLink: "https://ipfs.io/ipfs/QmYaGRJSZV8FjjD9Bd7BAQ5KevCQfbG3mU4yb2PNQAr1Vh/A wonderful.pdf",
+        status: "Open"
+      }
+    ]
```

Comment:
Why is this hardcoded?

Reply:
the last time I did this the data was still not there, I will update asap 👍🏼

Reply:
Still hardcoded? @bukandicki